### PR TITLE
London class 7 - Ionut Tripa - Git & Github - week 1

### PR DESCRIPTION
**Pull request by iTrip23**
# Your Details

Ionut Tripa
London
Johnny Tripa

# Homework Details

Git & Github
Week 1

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -92,22 +92,22 @@ A place for us to store and share our code.
...
 
-To record what changes you made to the code
+To write a short description of the changes that you have made.
 
 ## Question 18
 
 Where can I look to find help with Git?
 
-The Git Cheatsheet!
+https://syllabus.codeyourfuture.io/git/cheatsheet/
```
Comment (Trobinsonshoots) [inline homework.md:113]: Nice that you added the codeyourfuture link to the Git cheatsheet.

File: homework.md [hunk 2]
```diff
@@ -4,83 +4,83 @@
...
 
 }
-```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+JavaScript is a scripting or programming language that allows you to implement complex features on web pages — every time a web page does more than just sit there and display static information for you to look at — displaying timely content updates, interactive maps, animated 2D/3D graphics, scrolling video jukeboxes, etc. — you can bet that JavaScript is probably involved.
```
Comment (meganlouiseh) [inline homework.md:45]: I love this description of what JavaScript is!

**Review by meganlouiseh — COMMENTED** [review]
I have finished my review :)

**Comment by Jpsi** [comment]
Well done!

I like that you've used a descriptive title for your PR.

Tip: Try using your own words to describe what JavaScript means. You can copy the MDN definition, but the former will allow you to master it quicker.

**Pull request closed without merging**